src/lib/undux/effects/updateFeedList.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+import type { Effects, Store } from 'undux'
+
+import userStorage from '../../gundb/UserStorage'
+import type { State } from '../GDStore'
+
+const updateFeedList: Effects<State> = (store: Store) => {
+  store.on('requestFeeds').subscribe(async requestFeeds => {
+    if (requestFeeds) {
+      const feeds = await userStorage.getStandardizedFeed(10, false)
```

Comment:
since we are getting all the feed we should remove params so is less confusing
```suggestion
      const feeds = await userStorage.getStandardizedFeed()
```